picard/util/webbrowser2.py

```diff
@@ -35,6 +37,12 @@ def open(url):
         webbrowser.open(url)
     except webbrowser.Error as e:
         QtWidgets.QMessageBox.critical(None, _("Web Browser Error"), _("Error while launching a web browser:\n\n%s") % (e,))
+    except TypeError as e:
+        if version_info.major == 3 and version_info.minor == 7 and version_info.micro == 0:
+            # See https://bugs.python.org/issue31014, webbrowser.open doesn't
+            # work on 3.7.0.
+            log.info("Working around https://bugs.python.org/issue31014 - URLs might not be opened in the correct browser")
+            webbrowser.open(url)
```

Comment:
Do i miss something here ? The workaround is to call `webbrowser.open(url)` a second time ??

Reply:
Yes. The second time `webbrowser.open` is called, the on-demand initialization in https://github.com/python/cpython/blob/7dfbd4967121ef6ecabe3fd53a430fd949f405fd/Lib/webbrowser.py#L72 is not executed.